bin/analyze.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
To make the script more cross-platform, perhaps the `#!/usr/bin/env sh` should be used here?

Reply:
Will that not conflict when [using multiple arguments](https://unix.stackexchange.com/questions/399690/multiple-arguments-in-shebang/399702#399702)?

Reply:
Checked on my machine, no problems with passing several arguments to the script - `$1` and `$2` work as intended

Reply:
Plus I am using the `#!/usr/bin/env sh` in my Docker image, and it works there as well